smoke-tests/smoke-tests.gradle

```diff
@@ -38,6 +38,25 @@ test {
   //In addition to that we disable them by default on local machines
   enabled = enabled && (System.getenv("CI") != null || findProperty('runSmokeTests'))
 
+  def suites = [
+    "glassfishAndJetty": ["**/GlassFishSmokeTest.*", "**/JettySmokeTest.*"],
```

Comment:
Sometimes the simplest solution is the best :D Why these buckets specifically?

Reply:
No particular reason. It shouldn't really matter which way these are grouped as long as suite run time is lower than longest build step.